.github/workflows/delete-v2-review-app.yml

```diff
@@ -19,34 +22,55 @@ jobs:
       - name: Set Environment variables
         run: |
           terraform_version=$(awk '/{/{f=/^terraform/;next}f' terraform/aks/provider.tf | grep -o [0-9\.]*)
+          echo "TERRAFORM_VERSION=$terraform_version" >> $GITHUB_ENV
+
           PR_NUMBER=${{ github.event.pull_request.number }}
           TF_STATE_FILE=pr-$PR_NUMBER.tfstate
           echo "DEPLOY_ENV=review_aks-$PR_NUMBER" >> $GITHUB_ENV
           echo "PR_NUMBER=$PR_NUMBER" >> $GITHUB_ENV
           echo "TF_STATE_FILE=$TF_STATE_FILE" >> $GITHUB_ENV
 
-          pr_state_file=$(az storage blob list -c att-tfstate \
-            --connection-string "${{ secrets.AZURE_STORAGE_CONNECTION_STRING }}" \
-            --prefix $TF_STATE_FILE --query "[].name" -o tsv)
+          tf_vars_file=terraform/aks/workspace_variables/review_aks.tfvars.json
+          echo "KEY_VAULT_NAME=$(jq -r '.key_vault_name' "${tf_vars_file}")" >> $GITHUB_ENV
+          echo "KEY_VAULT_APP_SECRET_NAME=$(jq -r '.key_vault_app_secret_name' "${tf_vars_file}")" >> $GITHUB_ENV
+          echo "KEY_VAULT_INFRA_SECRET_NAME=$(jq -r '.key_vault_infra_secret_name' "${tf_vars_file}")" >> $GITHUB_ENV
 
-          if [ -n "$pr_state_file" ]; then echo "TF_STATE_EXISTS=true" >> $GITHUB_ENV; fi;
+          tf_backend_vars_file=terraform/aks/workspace_variables/review_aks_backend.tfvars
+          STORAGE_ACCOUNT_RG="s189t01-att-rv-rg"
```

Comment:
maybe use the same format as STORAGE_ACCOUNT_NAME below? i.e. echo "STORAGE_ACCOUNT_RG=${RESOURCE_NAME_PREFIX}... Would need to be after line 42